crates/ibc/Cargo.toml

```diff
@@ -17,7 +17,7 @@ description  = """
 all-features = true
 
 [features]
-default = ["std"]
+default = ["std", "disable_upgrade_client"]
```

Comment:
we should instead have the feature be `upgraded_client`, and _if enabled_, we provide the unfinished implementation. This is a subtle but necessary change: `no_std` users will build with `default-features=false`, which would "enable" the upgrade client code.

Shouldn't be a big change; feature checks just need to be flipped
```rust
cfg!(not(feature = "upgrade_client"))
```

Reply:
Done [83de030](https://github.com/cosmos/ibc-rs/pull/349/commits/83de030a251baf740659d3c8cb1612ad01cc2b81)